examples/platforms/efr32mg12/flash.c

```diff
@@ -32,68 +32,92 @@
  *   storage for the EFR32 platform using the Silabs Nvm3 interface.
  */
 
+#include "openthread-core-efr32-config.h"
 #include <openthread/config.h>
-#include <openthread/platform/settings.h>
 
-#if !OPENTHREAD_CONFIG_PLATFORM_FLASH_API_ENABLE
+#if OPENTHREAD_CONFIG_PLATFORM_FLASH_API_ENABLE // Defaults to OT NV system
```

Comment:
Comment is misleading, the default for efr32 is to use NVM3 (default value of macro in config.h is 0 for efr32xx)

Reply:
Fixed it. Thank you.